index.html

```diff
@@ -144,18 +144,51 @@
       <div class="card__prompt">
         <div>choose your weapon</div>
         <div>
-          <a class="card__back btn btn--small" 
+          <a class="card__back btn btn--small"
              href="#"
              previous="card--class">
             <span class="btn__prompt">&lt;</span>
             <span class="btn__text">Back</span>
-          </a>          
+          </a>
         </div>
       </div>
 
```

Comment:
can you add a comment "chunk" 

<!-- ******************************************************* -->
<!--                           WEAPSONS                               -->
<!-- ******************************************************* -->